Volontulo/app/src/main/java/com/stxnext/volontulo/model/Volunteer.java

```diff
@@ -0,0 +1,41 @@
+package com.stxnext.volontulo.model;
+
+import android.support.annotation.DrawableRes;
+
+public class Volunteer {
+    private String name;
```

Comment:
This same suggestion as a Offer class/form. Names of class properties should be realated with names of fields in the form: firstName, lastName, image, phoneNo: http://volontuloapp.org/me